### PR TITLE
Input component

### PR DESCRIPTION
**Pull request by iameru**
Here we create the story and the input.

### CODE REVIEW

**Comment by iameru** [comment]
implemented #3 . please merge

File: src/components/Input.tsx [hunk 1]
```diff
@@ -1,8 +1,17 @@
 import { useRef } from "react";
-
-export function Input() {
+type props = {
+  type: string;
+};
+export function Input({ type }: props) {
   const inputRef = useRef();
```
Comment (dnmct) [inline src/components/Input.tsx:8]: you should not need this
wrap everything with a label to focus on click

File: src/components/Input.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import { useRef } from "react";
+import { EnvelopeIcon, LockClosedIcon } from "@heroicons/react/24/outline";
```
Comment (dnmct) [inline src/components/Input.tsx:2]: according to design icons should be solid

**Review by dnmct — CHANGES_REQUESTED** [review]

File: src/components/Input.tsx (hunk 2 above)
Comment (iameru) [reply src/components/Input.tsx]: solved in 6696ccd

File: src/components/Input.tsx (hunk 1 above)
Comment (iameru) [reply src/components/Input.tsx]: solved in 6696ccd

**Pull request merged**